lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -8679,38 +8679,7 @@ namespace Js
         // The correct flag value is CallFlags_Value but we pass CallFlags_None in compat modes
         CallFlags flags = CallFlags_Value;
 
-        if (pArr)
```

Comment:
>pArr [](start = 12, length = 4)

It would be interesting to see what kind of perf impact this change has.

Reply:
Ran perftester and perf seems to be flat.